docs/ARCHITECTURE.md

```diff
@@ -224,6 +225,35 @@ The IncrementalRepairTask is the class that will perform the incremental repair
 
 [i96]: https://github.com/Ericsson/ecchronos/issues/96
 
+## Ecchronos as an Agent
+The default implementation of EcChronos was designed to be a Side-car for Cassandra, the new approach is to enable EcChronos instances to be responsible for more than one Cassandra node, at the rack, datacenter, or even entire cluster level.
```

Comment:
"ecChronos", nothing else. It's like spelling iPhone...

Reply:
Done